### PR TITLE
refactor: Fix ethereum deploy script

### LINKED ISSUE #87: Fix ethereum deployment script
For now deployment script works only for local ethereum node

### PR DESCRIPTION
**Pull request by koinov**
Deployment procedure update
Resolves #87

### CODE REVIEW

File: ethereum/src/libraries/Environment.sol [hunk 1]
```diff
@@ -1,5 +1,6 @@
 pragma solidity ^0.8.24;
 
+/*
```
Comment (mertwole) [inline ethereum/src/libraries/Environment.sol:3]: Remove these addresses that you've commented out

File: ethereum/src/Relayer.sol [hunk 2]
```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.8.24;
 import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
 import {IVerifier} from "./interfaces/IVerifier.sol";
 import {IRelayer} from "./interfaces/IRelayer.sol";
-import  "./libraries/Environment.sol";
+//import  "./libraries/Environment.sol";
```
Comment (mertwole) [inline ethereum/src/Relayer.sol:6]:
```suggestion
```

File: ethereum/test/MessageQueue.t.sol [hunk 3]
```diff
@@ -35,34 +35,34 @@ contract MessageQueueTest is TestHelper {
 
     function setUp() public override {
```
Comment (mertwole) [inline ethereum/test/MessageQueue.t.sol:36]: Can't you use `setUp` from `TestHelper.t.sol`?

File: ethereum/client/src/tests.rs [hunk 4]
```diff
@@ -137,7 +135,68 @@ where
...
+
+    pending_tx
+        .get_receipt()
+        .await
+        .map_err(|_| Error::ErrorWaitingTransactionReceipt)?;
+
+    let nonce = provider
+        .get_transaction_count(signer_address)
+        .await
+        .map_err(|_| Error::ErrorInHTTPTransport)?;
```
Comment (mertwole) [inline ethereum/client/src/tests.rs:164]: Move it to separate `fn`

Comment (mertwole) [reply ethereum/client/src/tests.rs:164]: And also - can `nonce` be managed by provider itself?

**Review by mertwole — CHANGES_REQUESTED** [review]

File: ethereum/client/src/tests.rs (hunk 4 above)
Comment (koinov) [reply ethereum/client/src/tests.rs:164]: Let's wait for alloy fix : https://github.com/alloy-rs/alloy/issues/1073

File: ethereum/test/MessageQueue.t.sol (hunk 3 above)
Comment (koinov) [reply ethereum/test/MessageQueue.t.sol:36]: done

File: ethereum/src/Relayer.sol (hunk 2 above)
Comment (koinov) [reply ethereum/src/Relayer.sol:6]: done

File: ethereum/src/libraries/Environment.sol (hunk 1 above)
Comment (koinov) [reply ethereum/src/libraries/Environment.sol:3]: done

**Review by mertwole — APPROVED** [review]

**Pull request merged**